--- pyp2rpm-3.3.0/pyp2rpm/utils.py~	2017-12-14 17:50:37.000000000 +0200
+++ pyp2rpm-3.3.0/pyp2rpm/utils.py	2018-02-04 14:47:29.799743953 +0200
@@ -163,7 +163,6 @@
     else:
         save_path = rpm_eval(macro)
         if not save_path:
-            logger.warn("rpm tools are missing, using default save path "
-                        "~/rpmbuild/.")
-            save_path = os.path.expanduser('~/rpmbuild')
+            logger.warn("rpm tools are missing, using current dir as save path.")
+            save_path = os.path.expanduser('.')
     return save_path
