--- pyp2rpm-1.0.1/pyp2rpm/settings.py~	2012-11-26 15:22:18.000000000 +0200
+++ pyp2rpm-1.0.1/pyp2rpm/settings.py	2013-03-17 03:10:31.762499704 +0200
@@ -5,7 +5,7 @@
 DEFAULT_METADATA_SOURCE = 'pypi'
 DEFAULT_TEMPLATE = 'fedora'
 DEFAULT_DISTRO = 'fedora'
-DEFAULT_PKG_SAVE_PATH = os.path.expanduser('~/rpmbuild/SOURCES/')
+DEFAULT_PKG_SAVE_PATH = os.path.expanduser('.')
 KNOWN_DISTROS = ['fedora', 'mageia']
 ARCHIVE_SUFFIXES = ['.tar', '.tgz', '.tar.gz', '.tar.bz2', '.gz', '.bz2', '.zip', '.egg']
 EXTENSION_SUFFIXES = ['.c', '.cpp']
