--- pyp2rpm-2.0.0/pyp2rpm/settings.py~	2016-03-11 23:24:36.000000000 +0200
+++ pyp2rpm-2.0.0/pyp2rpm/settings.py	2016-03-11 23:25:04.213223094 +0200
@@ -4,8 +4,8 @@
 DEFAULT_ADDITIONAL_VERSION = '3'
 DEFAULT_PKG_SOURCE = 'pypi'
 DEFAULT_METADATA_SOURCE = 'pypi'
-DEFAULT_TEMPLATE = 'fedora'
-DEFAULT_DISTRO = 'fedora'
+DEFAULT_TEMPLATE = 'pld'
+DEFAULT_DISTRO = 'pld'
 DEFAULT_PKG_SAVE_PATH = os.path.expanduser('.')
 KNOWN_DISTROS = ['fedora', 'mageia', 'pld']
 ARCHIVE_SUFFIXES = ['.tar', '.tgz', '.tar.gz', '.tar.bz2', '.gz', '.bz2', '.zip', '.egg']
