--- pyp2rpm-3.3.0/pyp2rpm/package_data.py~	2018-02-04 14:48:00.000000000 +0200
+++ pyp2rpm-3.3.0/pyp2rpm/package_data.py	2018-02-04 14:49:12.598463752 +0200
@@ -2,6 +2,7 @@
 import time
 import locale
 import logging
+import os
 
 from pyp2rpm import version
 from pyp2rpm import utils
@@ -90,15 +90,9 @@
     def get_changelog_date_packager(self):
         """Returns part of the changelog entry, containing date and packager.
         """
-        try:
-            packager = subprocess.Popen(
-                'rpmdev-packager', stdout=subprocess.PIPE).communicate(
-                )[0].strip()
-        except OSError:
-            # Hi John Doe, you should install rpmdevtools
-            packager = "John Doe <john@doe.com>"
-            logger.warn("Package rpmdevtools is missing, using default "
-                        "name: {0}.".format(packager))
+        packager = "John Doe <john@doe.com>"
+        if os.has_key('USER'):
+            packager = os.environ['USER']
         with utils.c_time_locale():
             date_str = time.strftime('%a %b %d %Y', time.gmtime())
         encoding = locale.getpreferredencoding()
