diff -urNp -x '*.orig' pyp2rpm-3.3.2.org/pyp2rpm/package_data.py pyp2rpm-3.3.2/pyp2rpm/package_data.py
--- pyp2rpm-3.3.2.org/pyp2rpm/package_data.py	2018-03-21 14:54:40.000000000 +0100
+++ pyp2rpm-3.3.2/pyp2rpm/package_data.py	2021-03-06 21:01:41.449600641 +0100
@@ -2,6 +2,7 @@ import subprocess
 import time
 import locale
 import logging
+import os
 
 from pyp2rpm import version
 from pyp2rpm import utils
@@ -89,15 +90,9 @@ class PackageData(object):
     def get_changelog_date_packager(self):
         """Returns part of the changelog entry, containing date and packager.
         """
-        try:
-            packager = subprocess.Popen(
-                'rpmdev-packager', stdout=subprocess.PIPE).communicate(
-                )[0].strip()
-        except OSError:
-            # Hi John Doe, you should install rpmdevtools
-            packager = "John Doe <john@doe.com>"
-            logger.warn("Package rpmdevtools is missing, using default "
-                        "name: {0}.".format(packager))
+        packager = "John Doe <john@doe.com>"
+        if os.has_key('USER'):
+            packager = os.environ['USER']
         with utils.c_time_locale():
             date_str = time.strftime('%a %b %d %Y', time.gmtime())
         encoding = locale.getpreferredencoding()
